bemuse/src/app/ui/MusicInfoTabStats.scss

```diff
@@ -25,7 +25,6 @@
       float: right;
       text-align: right;
       font-size: 1.5em;
-      line-height: 1;
```

Comment:
This affects the layout and is intended. Please revert this change.

Reply:
line-height: 1em; was specified after two lines. So I thought that there is a duplication. But it is correct to specify a number with no unit, so fixed on 4bd7730b.